js/src/framework.ts

```diff
@@ -376,7 +411,9 @@ export async function runEvaluator(
           meta({ scores: scoreMetadata });
         }
 
-        rootSpan.log({ scores, metadata });
+        // Note: We're asserting here that any object can be cast as Record<string, unknown>, which should generally
```

Comment:
This whole thing should not be necessary if we type `metadata` as `Record<string, unknown>` on line 341.